Directory.Packages.props

```
@@ -5,10 +5,9 @@
     <PackageVersion Include="JetBrains.Annotations" Version="2020.1.0" />
     <PackageVersion Include="JustEat.HttpClientInterception" Version="3.0.0" />
     <PackageVersion Include="MartinCostello.Logging.XUnit" Version="0.1.0" />
-    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="3.1.5" />
-    <PackageVersion Include="Microsoft.AspNetCore.TestHost" Version="3.1.5" />
-    <PackageVersion Include="Microsoft.IdentityModel.Protocols" Version="5.6.0" />
-    <PackageVersion Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageVersion Include="Microsoft.AspNetCore.Mvc.Testing" Version="5.0.0-rc.1.*" />
+    <PackageVersion Include="Microsoft.AspNetCore.TestHost" Version="5.0.0-rc.1.*" />
+    <PackageVersion Include="Microsoft.IdentityModel.Protocols" Version="6.7.1" />
```

Downgrade to match the version 5.0 will ship with.

```suggestion
    <PackageVersion Include="Microsoft.IdentityModel.Protocols" Version="6.6.0" />
```